task2/task2.2.js

```diff
@@ -11,111 +21,125 @@ app.use(function(req, _res, next) {
     next();
 });
 
-class User {
-    constructor(id, login, password, age, isDeleted = false) {
-        this.id = id ?? get_uuid();
-        this.login = login;
-        this.password = password;
-        this.age = age;
-        this.isDeleted = isDeleted;
+const predefined_users = [
+    {login: 'user1', password: '123qwe', age: 10},
+    {login: 'user2', password: '456ert', age: 100},
+    {login: 'user3', password: 'asd657', age: 50}
+];
+
+const User = sequelize.define('User', {
+    id: {
+        type: DataTypes.UUID,
+        allowNull: false,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+    },
+    login: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    password: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    age: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+    },
+    isDeleted: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
     }
-}
+}, {
+    timestamps: false
+});
 
 class Users {
     constructor() {
-        this.users = [];
-    }
-
-    find(id) {
-        return id && this.users.find(u => u.id === id);
+        sequelize.sync({});
+        
+        User.bulkCreate(predefined_users);
     }
 
-    get(id) {
-        const user = this.find(id);
-        if (user && user.isDeleted !== true) {
-            const retUser = { ...user };
-            delete retUser.isDeleted;
-            return retUser;
-        }
-        return null;
+    async get(id) {
+        return id && User.findAll({
+            attributes: {exclude: ['isDeleted']},
+            where: {
+                id,
+                isDeleted: {[Op.not]: true}
+            }
+        });

```

Comment:
it's better to create some userRepository and encapsulate all interactions with ORM. If the db is needed to be changed, you will be needed to change business logic layer as well